### ISSUE TITLE
[wish] Kartenbeschreibungen

### ISSUE THREAD

**Issue by Spikeone** [Return-To-The-Roots/s25client#64]
Detailliertere Kartenbeschreibungen.
Beispiel:

http://www1.picfront.org/picture/ImWwGPn4Y/img/DasAmulett.png
Name: Das Amulett
Autor:Blue Byte
Größe:192x176
Spieler:6
Zeit bis Feindkontakt:Lang
Bauplatz:Viel
Ausgeglichenheit:Ausgeglichen
Stein:Ausreichend
Beschreibung:Eine Karte für die man etwas mehr Zeit benötigt. Sie ist sehr gut für
Spiele bei denen Teams mit Drei gegen Drei Spielern spielt. Durch
ihren Symmetrischen Aufbau ist "Das Amulett" sehr ausgeglichen da-
durch entscheidet lediglich spielerische Überlegenheit über den
Ausgang des Spiels.

https://bugs.launchpad.net/s25rttr/+bug/367915

**Comment by MarcusSt** [comment]
This would require a different (or modified) map format. The original map format has no space for descriptions at all. For a modified format, it would be great to be able to run it with S2.

**Comment by Spikeone** [comment]
Why not Lua? Display the text defined in variable MAPINFO, if empty or no file: map size, player

**Comment by Flow86** [comment]
would be a possibility - or simply a txt file for the text displayed

**Comment by Spikeone** [comment]
Yeah, my First idea was to load a map name.txt but with Lua that makes 3 files for a map in some cases, that's why I think a Lua implementation would be handy

**Comment by MarcusSt** [comment]
Spike verified yesterday that appending new data to existing map files does not break S2, so it would be possible to modify the original map format and still be able to play the maps in S2.

**Comment by Spikeone** [comment]
I Think S2 Stops Reading the map file when the FF at the end is reached. Is it possible to add rttr codes/header maybe?

**Comment by Flamefire** [comment]
This again occurred in #415
I think we should move to an own map format. Who needs to play RTTR maps with S2 at the current state of RTTR?
The new map format should be able to transparently load S2 maps (new data will be default/empty) but also allow more things: Using terrains and objects from all gfx sets. Include map description and campaign settings (force player faces, alliances, AIs, addons...) and probably even the lua script itself. We can use the approach done by other games: Package this 3 files (map, description, lua file) into a zip archiv so we have a single file, instead of 3 or a blown up map format.
